storagenode/piecestore/endpoint.go

```diff
@@ -59,11 +59,56 @@ type Config struct {
 	MaxConcurrentRequests int           `help:"how many concurrent requests are allowed, before uploads are rejected." default:"6"`
 	OrderLimitGracePeriod time.Duration `help:"how long after OrderLimit creation date are OrderLimits no longer accepted" default:"1h0m0s"`
 	RetainTimeBuffer      time.Duration `help:"allows for small differences in the satellite and storagenode clocks" default:"1h0m0s"`
+	RetainStatus          RetainStatus  `help:"allows configuration to enable, disable, or test retain requests from the satellite" default:"disabled"`
 
 	Monitor monitor.Config
 	Sender  orders.SenderConfig
 }
 
+// RetainStatus is a type defining the enabled/disabled status of retain requests
+type RetainStatus int
```

Comment:
This is a status, we don't expect to do any math operations with it, so I think that `uint32` would be better.